src/RapidezServiceProvider.php

```diff
@@ -128,6 +128,7 @@ protected function bootPublishables(): self
 
             $this->publishes([
                 __DIR__ . '/../resources/lang' => resource_path('lang/vendor/rapidez'),
```

Comment:
Should we move this? As language files where moved with Laravel 9, see: https://laravel.com/docs/9.x/upgrade#the-lang-directory and all examples here: https://laravel.com/docs/11.x/packages#language-files refer to the lang directory directly.

Reply:
We can, for sure. The only issue is that it makes the publish command a bit more annoying if the json file and the php files are in the same folder, as the structure you should get after publishing is different. The way that the php translations work with the namespace is different to how the json files work (for some reason) so you get a discrepancy like this:

```
(what it looks like in the core)
/lang
    /en
        frontend.php
    /nl
        frontend.php
    nl.json
```

```
(in project after publishing, should look like this)
/lang
    /vendor
        /rapidez
            /en
                frontend.php
            /nl
                frontend.php
    nl.json
```